New package: WorldTimeAPI v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: WorldTimeAPI
- Repository: https://github.com/raphasampaio/WorldTimeAPI.jl
- Created by: @raphasampaio
- Version: v0.1.0
- Commit: 20d3d29ae1ed0e6c498675c8a6f3f422e75b3bcf
- Reviewed by: @raphasampaio
- Reference: https://github.com/raphasampaio/WorldTimeAPI.jl/commit/20d3d29ae1ed0e6c498675c8a6f3f422e75b3bcf#commitcomment-144455286
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68d0f187aa1b5fcce6dede115bdab8348fda596a303d8b405dcc67c675d99ac0fcf5d10e6a98575cb8dbd28f70ec61f25092a07b617401d93d341b86fbd1ca4fe54d222533a97f7cbc2d722039ad3a0f48ef1a89310c27896b415d981305c65e613764004423fc3089c527248ff689a17d1dde96c1688ed0dedee00e437909a566284d043d5e1134ed5f758b82573e84a345b12e35c7121c10516816ad3ee31394da82a310d4423214e56904320a8ab21 -->

goerz commented:
Thank you for submitting your package! However, this doesn't quite seem ready for registration. The package only seems to contain a single function and is missing documentation. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), the README should contain a description of the package's purpose and a small but complete usage example.

goerz commented:
[noblock] Great, thanks! 

Note that you have to edit your comment to contain `[noblock]`, or it will prevent the merge

JuliaRegistrator committed:
New package: WorldTimeAPI v0.1.0

UUID: f0211111-1d23-4195-bb37-87ab15ed0197
Repo: https://github.com/raphasampaio/WorldTimeAPI.jl.git
Tree: d10999eff43fccf4d06050d4b2f30f9f215bd379

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
